src/test/lib/utils/plugin_manager_test.cpp

```diff
@@ -91,6 +91,15 @@ TEST_F(PluginManagerTest, LoadingNonInstantiableLibrary) {
   EXPECT_THROW(pm.load_plugin(build_dylib_path("libhyriseTestNonInstantiablePlugin")), std::exception);
 }
 
+TEST_F(PluginManagerTest, LoadingDifferentPlugins) {
+  auto& pm = Hyrise::get().plugin_manager;
+  auto& plugins = get_plugins();
+
+  EXPECT_EQ(plugins.size(), 0u);
+  pm.load_plugin(build_dylib_path("libhyriseTestPlugin"));
+  pm.load_plugin(build_dylib_path("libhyriseMVCCDeletePlugin"));
+}
```

Comment:
This test is considered successful if the loading of the second plugin does not crash, right? For an outsider the test might become clearer with another `EXPECT_EQ(get_plugins().size(), 2u);` in the end. What do you think?